src/server/user-controller.js

```diff
@@ -98,9 +96,9 @@ class UserController {
     this.requestLimitRecordService = requestLimitRecordService;
     this.passwordResetRequestService = passwordResetRequestService;
 
-    this.passwordResetRateLimitOptions = {
-      maxRequests: PASSWORD_RESET_REQUEST_LIMIT,
-      expiresInMs: PASSWORD_RESET_REQUEST_TIME_WINDOW_IN_MS,
+    this.antiBruteForceRequestLimitOptions = {
+      maxRequests: ANTI_BRUTE_FORCE_MAX_REQUESTS,
+      expiresInMs: ANTI_BRUTE_FORCE_EXPIRES_IN_MS,
       service: this.requestLimitRecordService
```

Comment:
Actually is the `requestLimitRecordService` used anymore in this case? 🤔

Reply:
Yes, it is used by the middleware that is configured using these options.

Reply:
BUUT, good catch: I can delete 2 of the 3 service methods! Will do.

Reply:
Done!